installer_dir/hosts_list.yaml

```diff
@@ -0,0 +1,129 @@
+- checked_in_at: '2020-07-22T12:18:54'
+  cluster_id: 51724237-025b-4693-993f-49d321ec2ea5
+  created_at: '2020-07-22T12:07:53'
+  discovery_agent_version: quay.io/ocpmetal/agent:latest
+  hardware_info: '{"block_devices":[{"device_type":"loop","fstype":"squashfs","major_device_number":7,"name":"loop0","size":746217472},{"device_type":"rom","fstype":"iso9660","major_device_number":11,"name":"sr0","removable_device":1,"size":822083584},{"device_type":"disk","major_device_number":252,"name":"vda","size":21474836480}],"cpu":{"architecture":"x86_64","cpu_mhz":2095.076,"cpus":4,"model_name":"Intel(R)
+    Xeon(R) Gold 6152 CPU @ 2.10GHz","sockets":4,"threads_per_core":1},"memory":[{"free":15661727744,"name":"Mem","shared":1127391232,"total":17378611200,"used":1716883456},{"name":"Swap"}],"nics":[{"cidrs":[{"ip_address":"192.168.126.11","mask":24}],"mac":"52:54:00:42:1e:8d","mtu":1500,"name":"eth0","state":"BROADCAST,MULTICAST,UP,LOWER_UP"},{"cidrs":[{"ip_address":"192.168.140.133","mask":24}],"mac":"52:54:00:ca:7b:16","mtu":1500,"name":"eth1","state":"BROADCAST,MULTICAST,UP,LOWER_UP"},{"cidrs":[{"ip_address":"10.88.0.1","mask":16}],"mac":"8a:97:25:6c:c0:61","mtu":1500,"name":"cni-podman0","state":"NO-CARRIER,BROADCAST,MULTICAST,UP"}]}'
+  href: /api/assisted-install/v1/clusters/51724237-025b-4693-993f-49d321ec2ea5/hosts/26cc8e92-0f5a-4d05-93e1-0c55667a723b
```

Comment:
What is all this hard coded uuids?

Reply:
This is used for local testing: running ignition-kubeconfig container on your laptop. This file contains examples of output of list-hosts api from bm-inventory

Reply:
So this file should be named appropriately, so we know it is used for testing

Reply:
fixed